crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,8 +285,9 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
-          file.to_s =~ /knife\/databags\/crowbar/
+        !file.basename.to_s.match(
+          /(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)|(^swift\.json$)/
+        ) && file.to_s =~ /knife\/databags\/crowbar/
```

Comment:
How about actually loading the file and check if it has an "attributes" and a "deployment" section? That way we can be pretty sure it's a proposal.